run_robot_test.sh

```diff
@@ -114,6 +114,34 @@ case "$(uname -s)" in
         ;;
 esac
 
+## if we have yq installed
+if command -v yq &> /dev/null
+then
+    echo "INFO: we found a yq executable"
+
+    ## get the user, pass and API hostname for OpenShift
+    oc_user=$(yq  e '.OCP_ADMIN_USER.USERNAME' ${TEST_VARIABLES_FILE})
+    oc_pass=$(yq  e '.OCP_ADMIN_USER.PASSWORD' ${TEST_VARIABLES_FILE})
+    oc_host=$(yq  e '.OCP_API_URL' ${TEST_VARIABLES_FILE})
+
+    ## do an oc login here
+    oc login "${oc_host}" --username "${oc_user}" --password "${oc_pass}" --insecure-skip-tls-verify=true
```

Comment:
Is this required for test execution? I assume this is for the robot-openshiftcli use and, if so, shouldn't it perform it's own authentication during instantiation that would be contained to that running instance?

Reply:
yeah, it's not the best place for oc login. But we need it to make the test running. We'll remove it as soon as the robot-openshiftcli lib will handle the authentication itself